ci/build-conda.sh

```diff
@@ -60,7 +63,7 @@ if [[ gpu_only -ne 1 ]];then
     echo "================================"
     echo "  Starting CPU Build and Test"
     echo "================================"
-    build_test_cpu
+    #build_test_cpu
```

Comment:
Should we enable this?

Reply:
Oh yes totally. In the end, do we really need to do both? Why not just doing GPU tests because it will also do the CPU ones.